docs/faq.rst

```diff
@@ -1,6 +1,20 @@
 FAQ
 ===
 
+What happens if I rename my GitHub repository?
+----------------------------------------------
+
+If you have registered a tool or workflow from GitHub in Dockstore the
```

Comment:
I suggest starting off this paragraph with with a version of of your second sentence -- it flows better from the title. Either that, or change "If" here to "When".

Reply:
Changed If to When